podcasts/lock-the-quill.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 rss_url: https://feeds.buzzsprout.com/2074298.rss
-topics: Technology
+topics: Mechanical Engineering
```

Comment:
This whole podcast was not loaded - i think we can't parse the rss link. It's missing on prod as well